tests/ctest/test_utility.c

```diff
@@ -96,6 +122,7 @@ int main (int argc, char ** argv)
 	test_elektraLskip ();
 	test_elektraRstrip ();
 	test_ElektraStrip ();
+	test_ElektraReplace ();
```

Comment:
Oh, I did not even notice this inconsistency. I already fixed it in my local copy of the repository.